economics/staking_protocol/main.tex

```diff
@@ -0,0 +1,355 @@
+\\\\
+Consequently, a two-phase subsidy model assists all would-be service-providers who don't happen to stake right at network launch but decide to do so at a later date. By addressing the temporal inequity of immediately decreasing issuance, we impede problematic centralization trends.
+
+\subsubsection{Risks to operational feasibility, incentives and pricing}
+
+As illustrated in figure 1, generic subsidy models comprise a high issuance at genesis and in the first few years of the network's existence. Setting the peak issuance (the maximum number of tokens minted per day) at a comparatively lower figure may address the following problems: 
+\begin{itemize}
+\item The greater the real-world value of the subsidy, the further from a fee-driven `reality' a service-provider's set-up may be in economic terms (for example, in terms of operational efficiency). When subsidies eventually run out, service-providers who are unable to adapt may cease operations, threatening general service availability or even reneging on existing commitments to users. Under a model with higher token issuance early on, these inefficient operations may control a substantial proportion of the circulating supply by this point. 
+\item If subsidies are worth significantly more than the typical sum of transaction fees earned in the same period, then the incentive to earn those fees may be blunted. This is particularly problematic for the protocol if the work required to collect subsidies does not align well with work required to earn fees, and the reliability of the latter impacts overall service quality. 
+\item High but decreasing subsidies may compel some service-providers to offer an unsustainably cheap service, with the intention of raising prices later once subsidies dry up. Any developers and their end-users that have become reliant on the service, but are unable to afford the higher price point, may see their application/business thrown into jeopardy. The existence of this risk is a serious friction for would-be network users planning an integration and justifying the associated upfront costs and technical dependency.
+\item In NuCypher's case, existing market prices for comparable services – key/secrets management and dynamic access control – are extremely low on a per-user basis. Profitability is generally achievable through high volume, which is a long-term endeavour. This business reality aggravates all three above issues described in this sub-section.
```

Comment:
This bullet point needs a reference and a bit of elaboration. Key management services are very cheap on a per-operation basis. On a per-user basis, they tend to also be cheap because developers rent a single master key and use it to generate keys for multiple users. The only exception are CloudHSMs, which are expensive and max out at 3,300 keys (again, this would probably only require you to rent a second HSM if you're using a unique key per user).